src/js/components/SidebarComponent.jsx

```diff
@@ -35,16 +34,6 @@ var SidebarComponent = React.createClass({
     };
   },
 
-  updateFilter: function (filterName, filter) {
-    var filters = Object.assign({}, this.state.filters, {
-      [filterName]: filter
-    });
-
-    this.setState({
-      filters: filters
-    }, this.props.onChange(filters));
-  },
-
```

Comment:
FTR: removing this without changing the way filterLabels are fetched from the query params broke the label filters. https://github.com/mesosphere/marathon-ui/pull/602